### PR TITLE
view BE product

### PR DESCRIPTION
**Pull request by quyensama**
## Related Tickets:
- [#45905](https://edu-redmine.sun-asterisk.vn/issues/45905)

## WHAT this PR does?
- view BE product

## Notes Impacted Areas:
- N/A.

## Attachments:
Screenshot from 2022-03-02 13-54-53

### CODE REVIEW

**Review by NamBZ — APPROVED** [review]

**Comment by quyensama** [comment]
ready

File: resources/views/dashboards/admin/products/create.blade.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    </form>
+                </div>
+                <!-- /.card-body -->
+            </div>
+            <!-- /.card -->
+        </div>
+    </div>
+</section>
+<!-- /.content -->
+@endsection
```
Comment (ghost) [inline resources/views/dashboards/admin/products/create.blade.php]: eof

File: resources/views/dashboards/admin/products/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                    </form>
+                </div>
+                <!-- /.card-body -->
+            </div>
+            <!-- /.card -->
+        </div>
+    </div>
+</section>
+<!-- /.content -->
+@endsection
```
Comment (ghost) [inline resources/views/dashboards/admin/products/edit.blade.php]: eof

**Comment by quyensama** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**